adoc/extensions/sycl_khr_dynamic_addrspace_cast.adoc

```diff
@@ -0,0 +1,102 @@
+[[sec:khr-dynamic-addrspace-cast]]
+= SYCL_KHR_DYNAMIC_ADDRSPACE_CAST
+
+This extension renames [code]#sycl::address_space_cast# to
+[code]#dynamic_addrspace_cast# to align with the [code]#static_addrspace_cast#
```

Comment:
Kind of pedantic, but I think "renames" is not the right word here because it implies that the old name is going away.  Maybe:

> This extension introduces `dynamic_addrspace_cast` with the same semantics as `sycl::address_space_cast` to align with ...

Reply:
I hope it _will_ be going away, but since it's not deprecated yet I've adopted your proposed wording in 49b99c7056b33142c61ec079b47a8c2ed878a4d4.